### ISSUE TITLE
Nice repository. Look forward to a more detailed README:)

### ISSUE THREAD

**Issue by HenryHZY** [ilkerkesen/frozen#3]
@ilkerkesen Do you have a plan to share a more detailed readme.md?

**Comment by ilkerkesen** [comment]
@HenryHZY yes, I'll need to make it working first :) I guess I'll complete the preliminary experiments on this weekend.

**Comment by HenryHZY** [comment]
> @HenryHZY yes, I'll need to make it working first :) I guess I'll complete the preliminary experiments on this weekend.

@ilkerkesen Hi, is the reproduction project going well?

**Comment by ilkerkesen** [comment]
@HenryHZY sorry for the late response. Actually I was considering to incorporate this approach into some another natural language generation task. However, I'm not happy with the caption generation performance. I can share some results later, but there are are two issues: (i) NFResNet produces NaN values and (ii) the default learning rate doesn't work. I was able to make it working somehow with ViT (not NFResNet) using smaller learning rate (5.0e-5), but the performance didn't satisfy me. I've got too much work right now, let me add some documentation for future researchers when I find time.
